api/_providers.ts

```diff
@@ -14,21 +16,33 @@ const { RPC_HEADERS } = getEnvs();
 export const providerCache: Record<string, providers.StaticJsonRpcProvider> =
   {};
 
+function getSvmRpcUrl(chainId: number): string {
+  const urls = getRpcUrlsFromConfigJson(chainId);
+  const publicNetwork = PUBLIC_NETWORKS[Number(chainId)];
```

Comment:
`chainId` is already scoped as a number - should be OK to drop this?

```suggestion
  const publicNetwork = PUBLIC_NETWORKS[chainId];
```

Reply:
Yep do it